client/verta/verta/_registry/models.py

```diff
@@ -42,9 +42,11 @@ def _create_element(self, msg):
         return RegisteredModel(self._conn, self._conf, msg)
 
     def with_workspace(self, workspace_name=None):
-        new_list = copy.deepcopy(self)
-        new_list._msg.workspace_name = workspace_name
-        return new_list
+        if workspace_name is not None:
+            new_list = copy.deepcopy(self)
+            new_list._msg.workspace_name = workspace_name
+            return new_list
+        return self
```

Comment:
Was this causing a bug? If not, I prefer we always `return new_list`, because other `LazyList.with_*()` methods always return a new object.

Suggesting revert:
```suggestion
                new_list = copy.deepcopy(self)
        new_list._msg.workspace_name = workspace_name
        return new_list
```

Reply:
None can't be assigned to _msg.workspace_name because _msg is immutable object, but I can copy an empty string to it. Used new_list._msg.workspace_name = '' here.